testing/src/streaming_service.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Tink-Rust Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+//! Testing server for streaming AEAD.
+
+use crate::proto;
+use tink::{utils::wrap_err, TinkError};
+use tink_testutil::SharedBuf;
+
+#[derive(Debug, Default)]
+pub struct StreamingAeadServerImpl {}
```

Comment:
(nit) for structs that you expect will never hold any fields, consider omitting the `{}`. Up to you really.